<?xml version="1.0" encoding="UTF-8" ?>
<testsuite failures="0" time="1.164" errors="18" skipped="0" tests="18" name="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/amd64"/>
    <property name="java.vm.version" value="20.1-b02"/>
    <property name="java.vm.vendor" value="Sun Microsystems Inc."/>
    <property name="java.vendor.url" value="http://java.sun.com/"/>
    <property name="path.separator" value=":"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.country" value="FR"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/home/jgribonvald/Travail/workspace-helios/esco/ext/grouper-api"/>
    <property name="java.runtime.version" value="1.6.0_26-b03"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="basedir" value="/home/jgribonvald/Travail/workspace-helios/esco/ext/grouper-api"/>
    <property name="java.endorsed.dirs" value="/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="surefire.real.class.path" value="/tmp/surefirebooter6789415486905512393.jar"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Sun Microsystems Inc."/>
    <property name="os.name" value="Linux"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/amd64/server:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/amd64:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="surefire.test.class.path" value="/home/jgribonvald/Travail/workspace-helios/esco/ext/grouper-api/target/test-classes:/home/jgribonvald/Travail/workspace-helios/esco/ext/grouper-api/target/classes:/opt/repo_maven/local/org/springframework/spring-context/2.5.6/spring-context-2.5.6.jar:/opt/repo_maven/local/aopalliance/aopalliance/1.0/aopalliance-1.0.jar:/opt/repo_maven/local/commons-logging/commons-logging/1.1.1/commons-logging-1.1.1.jar:/opt/repo_maven/local/org/springframework/spring-beans/2.5.6/spring-beans-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-core/2.5.6/spring-core-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-support/2.0.8/spring-support-2.0.8.jar:/home/jgribonvald/Travail/workspace-helios/esco/metier/esco-core/target/esco-core-2.6.5.jar:/opt/repo_maven/local/org/esco/grouperui/ext/esco-subject-extension/1.3/esco-subject-extension-1.3.jar:/opt/repo_maven/local/org/springframework/spring-aop/2.5.6/spring-aop-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-aspects/2.5.6/spring-aspects-2.5.6.jar:/opt/repo_maven/local/org/aspectj/aspectjrt/1.6.1/aspectjrt-1.6.1.jar:/opt/repo_maven/local/org/aspectj/aspectjweaver/1.6.1/aspectjweaver-1.6.1.jar:/opt/repo_maven/local/org/springframework/spring-context-support/2.5.6/spring-context-support-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-jdbc/2.5.6/spring-jdbc-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-tx/2.5.6/spring-tx-2.5.6.jar:/opt/repo_maven/local/commons-dbcp/commons-dbcp/1.2.2/commons-dbcp-1.2.2.jar:/opt/repo_maven/local/commons-pool/commons-pool/1.2/commons-pool-1.2.jar:/opt/repo_maven/local/org/apache/derby/derbytools/10.5.1.1/derbytools-10.5.1.1.jar:/opt/repo_maven/local/org/apache/derby/derby/10.5.1.1/derby-10.5.1.1.jar:/opt/repo_maven/local/org/apache/derby/derbyclient/10.5.1.1/derbyclient-10.5.1.1.jar:/opt/repo_maven/local/org/apache/derby/derbynet/10.5.1.1/derbynet-10.5.1.1.jar:/opt/repo_maven/local/net/sf/ehcache/ehcache/1.4.0/ehcache-1.4.0.jar:/opt/repo_maven/local/backport-util-concurrent/backport-util-concurrent/3.0/backport-util-concurrent-3.0.jar:/opt/repo_maven/local/commons-collections/commons-collections/3.2/commons-collections-3.2.jar:/opt/repo_maven/local/net/sf/jsr107cache/jsr107cache/1.0/jsr107cache-1.0.jar:/opt/repo_maven/local/javax/activation/activation/1.0.2/activation-1.0.2.jar:/opt/repo_maven/local/commons-configuration/commons-configuration/1.5/commons-configuration-1.5.jar:/opt/repo_maven/local/commons-lang/commons-lang/2.1/commons-lang-2.1.jar:/opt/repo_maven/local/commons-digester/commons-digester/1.8.1/commons-digester-1.8.1.jar:/opt/repo_maven/local/commons-beanutils/commons-beanutils/1.6/commons-beanutils-1.6.jar:/opt/repo_maven/local/commons-beanutils/commons-beanutils-core/1.7.0/commons-beanutils-core-1.7.0.jar:/opt/repo_maven/local/com/google/collections/google-collections/1.0-rc2/google-collections-1.0-rc2.jar:/opt/repo_maven/local/javax/xml/bind/jaxb-api/2.1/jaxb-api-2.1.jar:/opt/repo_maven/local/javax/xml/stream/stax-api/1.0-2/stax-api-1.0-2.jar:/opt/repo_maven/local/com/sun/xml/bind/jaxb-impl/2.1/jaxb-impl-2.1.jar:/opt/repo_maven/local/com/sun/xml/parsers/jaxp-ri/1.4.1/jaxp-ri-1.4.1.jar:/opt/repo_maven/local/javax/xml/parsers/jaxp-api/1.4.1/jaxp-api-1.4.1.jar:/opt/repo_maven/local/org/apache/ant/ant/1.7.1/ant-1.7.1.jar:/opt/repo_maven/local/org/apache/ant/ant-launcher/1.7.1/ant-launcher-1.7.1.jar:/opt/repo_maven/local/antlr/antlr/2.7.7/antlr-2.7.7.jar:/opt/repo_maven/local/asm/asm/1.5.3/asm-1.5.3.jar:/opt/repo_maven/local/asm/asm-attrs/1.5.3/asm-attrs-1.5.3.jar:/opt/repo_maven/local/asm/asm-util/1.5.3/asm-util-1.5.3.jar:/opt/repo_maven/local/bsh/bsh/1.3.0/bsh-1.3.0.jar:/opt/repo_maven/local/c3p0/c3p0/0.9.1.2/c3p0-0.9.1.2.jar:/opt/repo_maven/local/cglib/cglib/2.1_3/cglib-2.1_3.jar:/opt/repo_maven/local/commons-betwixt/commons-betwixt/0.8/commons-betwixt-0.8.jar:/opt/repo_maven/local/xml-apis/xml-apis/1.0.b2/xml-apis-1.0.b2.jar:/opt/repo_maven/local/commons-cli/commons-cli/1.1/commons-cli-1.1.jar:/opt/repo_maven/local/commons-discovery/commons-discovery/0.4/commons-discovery-0.4.jar:/opt/repo_maven/local/commons-io/commons-io/1.0/commons-io-1.0.jar:/opt/repo_maven/local/commons-math/commons-math/1.1/commons-math-1.1.jar:/opt/repo_maven/local/org/apache/ddlutils/ddlutils/1.0/ddlutils-1.0.jar:/opt/repo_maven/local/commons-codec/commons-codec/1.3/commons-codec-1.3.jar:/opt/repo_maven/local/commons-logging/commons-logging-api/1.0.4/commons-logging-api-1.0.4.jar:/opt/repo_maven/local/xerces/xercesImpl/2.0.2/xercesImpl-2.0.2.jar:/opt/repo_maven/local/dom4j/dom4j/1.6.1/dom4j-1.6.1.jar:/opt/repo_maven/local/junit/junit/4.6/junit-4.6.jar:/opt/repo_maven/local/stax/stax-api/1.0.1/stax-api-1.0.1.jar:/opt/repo_maven/local/oro/oro/2.0.8/oro-2.0.8.jar:/opt/repo_maven/local/org/hibernate/hibernate/3.2.6.ga/hibernate-3.2.6.ga.jar:/opt/repo_maven/local/javax/transaction/jta/1.0.1B/jta-1.0.1B.jar:/opt/repo_maven/local/com/jamonapi/jamon/2.0/jamon-2.0.jar:/opt/repo_maven/local/javax/servlet/servlet-api/2.3/servlet-api-2.3.jar:/opt/repo_maven/local/org/safehaus/jug/jug/2.0.0/jug-2.0.0-lgpl.jar:/opt/repo_maven/local/log4j/log4j/1.2.8/log4j-1.2.8.jar:/opt/repo_maven/local/javax/mail/mail/1.3.2/mail-1.3.2.jar:/opt/repo_maven/local/odmg/odmg/3.0/odmg-3.0.jar:/opt/repo_maven/local/p6spy/p6spy/1.3/p6spy-1.3.jar:/opt/repo_maven/local/org/opensymphony/quartz/quartz/1.6.1/quartz-1.6.1.jar:/opt/repo_maven/local/com/dawidweiss/invoker/1.0/invoker-1.0.jar:/opt/repo_maven/local/edu/internet2/middleware/grouper/grouper/1.6.2/grouper-1.6.2.jar:/opt/repo_maven/local/edu/internet2/middleware/grouper/subject/1.6.2/subject-1.6.2.jar:/opt/repo_maven/local/edu/internet2/middleware/grouper/morphString/1.0/morphString-1.0.jar:/opt/repo_maven/local/net/sf/ezmorph/ezmorph/1.0.6/ezmorph-1.0.6.jar:/opt/repo_maven/local/net/sf/json-lib/json-lib/2.3/json-lib-2.3-jdk15.jar:/opt/repo_maven/local/com/thoughtworks/xstream/xstream/1.3/xstream-1.3.jar:/opt/repo_maven/local/xpp3/xpp3_min/1.1.4c/xpp3_min-1.1.4c.jar:/opt/repo_maven/local/hsqldb/hsqldb/1.8.0.7/hsqldb-1.8.0.7.jar:/opt/repo_maven/local/mysql/mysql-connector-java/5.1.6/mysql-connector-java-5.1.6.jar:/opt/repo_maven/local/com/mycila/testing/mycila-testing-api/2.0-rc1/mycila-testing-api-2.0-rc1.jar:/opt/repo_maven/local/com/mycila/mycila-plugin/1.7/mycila-plugin-1.7.jar:/opt/repo_maven/local/com/mycila/mycila-log/1.2/mycila-log-1.2.jar:/opt/repo_maven/local/com/mycila/testing/plugins/mycila-testing-spring/2.0-rc1/mycila-testing-spring-2.0-rc1.jar:/opt/repo_maven/local/org/springframework/spring-test/2.5.6/spring-test-2.5.6.jar:/opt/repo_maven/local/com/mycila/testing/plugins/mycila-testing-mockito/2.0-rc1/mycila-testing-mockito-2.0-rc1.jar:/opt/repo_maven/local/org/mockito/mockito-core/1.7/mockito-core-1.7.jar:/opt/repo_maven/local/org/objenesis/objenesis/1.0/objenesis-1.0.jar:/opt/repo_maven/local/org/hamcrest/hamcrest-all/1.1/hamcrest-all-1.1.jar:"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="50.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="3.0.0-15-generic"/>
    <property name="user.home" value="/home/jgribonvald"/>
    <property name="user.timezone" value=""/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="1.6"/>
    <property name="user.name" value="jgribonvald"/>
    <property name="java.class.path" value="/home/jgribonvald/Travail/workspace-helios/esco/ext/grouper-api/target/test-classes:/home/jgribonvald/Travail/workspace-helios/esco/ext/grouper-api/target/classes:/opt/repo_maven/local/org/springframework/spring-context/2.5.6/spring-context-2.5.6.jar:/opt/repo_maven/local/aopalliance/aopalliance/1.0/aopalliance-1.0.jar:/opt/repo_maven/local/commons-logging/commons-logging/1.1.1/commons-logging-1.1.1.jar:/opt/repo_maven/local/org/springframework/spring-beans/2.5.6/spring-beans-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-core/2.5.6/spring-core-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-support/2.0.8/spring-support-2.0.8.jar:/home/jgribonvald/Travail/workspace-helios/esco/metier/esco-core/target/esco-core-2.6.5.jar:/opt/repo_maven/local/org/esco/grouperui/ext/esco-subject-extension/1.3/esco-subject-extension-1.3.jar:/opt/repo_maven/local/org/springframework/spring-aop/2.5.6/spring-aop-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-aspects/2.5.6/spring-aspects-2.5.6.jar:/opt/repo_maven/local/org/aspectj/aspectjrt/1.6.1/aspectjrt-1.6.1.jar:/opt/repo_maven/local/org/aspectj/aspectjweaver/1.6.1/aspectjweaver-1.6.1.jar:/opt/repo_maven/local/org/springframework/spring-context-support/2.5.6/spring-context-support-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-jdbc/2.5.6/spring-jdbc-2.5.6.jar:/opt/repo_maven/local/org/springframework/spring-tx/2.5.6/spring-tx-2.5.6.jar:/opt/repo_maven/local/commons-dbcp/commons-dbcp/1.2.2/commons-dbcp-1.2.2.jar:/opt/repo_maven/local/commons-pool/commons-pool/1.2/commons-pool-1.2.jar:/opt/repo_maven/local/org/apache/derby/derbytools/10.5.1.1/derbytools-10.5.1.1.jar:/opt/repo_maven/local/org/apache/derby/derby/10.5.1.1/derby-10.5.1.1.jar:/opt/repo_maven/local/org/apache/derby/derbyclient/10.5.1.1/derbyclient-10.5.1.1.jar:/opt/repo_maven/local/org/apache/derby/derbynet/10.5.1.1/derbynet-10.5.1.1.jar:/opt/repo_maven/local/net/sf/ehcache/ehcache/1.4.0/ehcache-1.4.0.jar:/opt/repo_maven/local/backport-util-concurrent/backport-util-concurrent/3.0/backport-util-concurrent-3.0.jar:/opt/repo_maven/local/commons-collections/commons-collections/3.2/commons-collections-3.2.jar:/opt/repo_maven/local/net/sf/jsr107cache/jsr107cache/1.0/jsr107cache-1.0.jar:/opt/repo_maven/local/javax/activation/activation/1.0.2/activation-1.0.2.jar:/opt/repo_maven/local/commons-configuration/commons-configuration/1.5/commons-configuration-1.5.jar:/opt/repo_maven/local/commons-lang/commons-lang/2.1/commons-lang-2.1.jar:/opt/repo_maven/local/commons-digester/commons-digester/1.8.1/commons-digester-1.8.1.jar:/opt/repo_maven/local/commons-beanutils/commons-beanutils/1.6/commons-beanutils-1.6.jar:/opt/repo_maven/local/commons-beanutils/commons-beanutils-core/1.7.0/commons-beanutils-core-1.7.0.jar:/opt/repo_maven/local/com/google/collections/google-collections/1.0-rc2/google-collections-1.0-rc2.jar:/opt/repo_maven/local/javax/xml/bind/jaxb-api/2.1/jaxb-api-2.1.jar:/opt/repo_maven/local/javax/xml/stream/stax-api/1.0-2/stax-api-1.0-2.jar:/opt/repo_maven/local/com/sun/xml/bind/jaxb-impl/2.1/jaxb-impl-2.1.jar:/opt/repo_maven/local/com/sun/xml/parsers/jaxp-ri/1.4.1/jaxp-ri-1.4.1.jar:/opt/repo_maven/local/javax/xml/parsers/jaxp-api/1.4.1/jaxp-api-1.4.1.jar:/opt/repo_maven/local/org/apache/ant/ant/1.7.1/ant-1.7.1.jar:/opt/repo_maven/local/org/apache/ant/ant-launcher/1.7.1/ant-launcher-1.7.1.jar:/opt/repo_maven/local/antlr/antlr/2.7.7/antlr-2.7.7.jar:/opt/repo_maven/local/asm/asm/1.5.3/asm-1.5.3.jar:/opt/repo_maven/local/asm/asm-attrs/1.5.3/asm-attrs-1.5.3.jar:/opt/repo_maven/local/asm/asm-util/1.5.3/asm-util-1.5.3.jar:/opt/repo_maven/local/bsh/bsh/1.3.0/bsh-1.3.0.jar:/opt/repo_maven/local/c3p0/c3p0/0.9.1.2/c3p0-0.9.1.2.jar:/opt/repo_maven/local/cglib/cglib/2.1_3/cglib-2.1_3.jar:/opt/repo_maven/local/commons-betwixt/commons-betwixt/0.8/commons-betwixt-0.8.jar:/opt/repo_maven/local/xml-apis/xml-apis/1.0.b2/xml-apis-1.0.b2.jar:/opt/repo_maven/local/commons-cli/commons-cli/1.1/commons-cli-1.1.jar:/opt/repo_maven/local/commons-discovery/commons-discovery/0.4/commons-discovery-0.4.jar:/opt/repo_maven/local/commons-io/commons-io/1.0/commons-io-1.0.jar:/opt/repo_maven/local/commons-math/commons-math/1.1/commons-math-1.1.jar:/opt/repo_maven/local/org/apache/ddlutils/ddlutils/1.0/ddlutils-1.0.jar:/opt/repo_maven/local/commons-codec/commons-codec/1.3/commons-codec-1.3.jar:/opt/repo_maven/local/commons-logging/commons-logging-api/1.0.4/commons-logging-api-1.0.4.jar:/opt/repo_maven/local/xerces/xercesImpl/2.0.2/xercesImpl-2.0.2.jar:/opt/repo_maven/local/dom4j/dom4j/1.6.1/dom4j-1.6.1.jar:/opt/repo_maven/local/junit/junit/4.6/junit-4.6.jar:/opt/repo_maven/local/stax/stax-api/1.0.1/stax-api-1.0.1.jar:/opt/repo_maven/local/oro/oro/2.0.8/oro-2.0.8.jar:/opt/repo_maven/local/org/hibernate/hibernate/3.2.6.ga/hibernate-3.2.6.ga.jar:/opt/repo_maven/local/javax/transaction/jta/1.0.1B/jta-1.0.1B.jar:/opt/repo_maven/local/com/jamonapi/jamon/2.0/jamon-2.0.jar:/opt/repo_maven/local/javax/servlet/servlet-api/2.3/servlet-api-2.3.jar:/opt/repo_maven/local/org/safehaus/jug/jug/2.0.0/jug-2.0.0-lgpl.jar:/opt/repo_maven/local/log4j/log4j/1.2.8/log4j-1.2.8.jar:/opt/repo_maven/local/javax/mail/mail/1.3.2/mail-1.3.2.jar:/opt/repo_maven/local/odmg/odmg/3.0/odmg-3.0.jar:/opt/repo_maven/local/p6spy/p6spy/1.3/p6spy-1.3.jar:/opt/repo_maven/local/org/opensymphony/quartz/quartz/1.6.1/quartz-1.6.1.jar:/opt/repo_maven/local/com/dawidweiss/invoker/1.0/invoker-1.0.jar:/opt/repo_maven/local/edu/internet2/middleware/grouper/grouper/1.6.2/grouper-1.6.2.jar:/opt/repo_maven/local/edu/internet2/middleware/grouper/subject/1.6.2/subject-1.6.2.jar:/opt/repo_maven/local/edu/internet2/middleware/grouper/morphString/1.0/morphString-1.0.jar:/opt/repo_maven/local/net/sf/ezmorph/ezmorph/1.0.6/ezmorph-1.0.6.jar:/opt/repo_maven/local/net/sf/json-lib/json-lib/2.3/json-lib-2.3-jdk15.jar:/opt/repo_maven/local/com/thoughtworks/xstream/xstream/1.3/xstream-1.3.jar:/opt/repo_maven/local/xpp3/xpp3_min/1.1.4c/xpp3_min-1.1.4c.jar:/opt/repo_maven/local/hsqldb/hsqldb/1.8.0.7/hsqldb-1.8.0.7.jar:/opt/repo_maven/local/mysql/mysql-connector-java/5.1.6/mysql-connector-java-5.1.6.jar:/opt/repo_maven/local/com/mycila/testing/mycila-testing-api/2.0-rc1/mycila-testing-api-2.0-rc1.jar:/opt/repo_maven/local/com/mycila/mycila-plugin/1.7/mycila-plugin-1.7.jar:/opt/repo_maven/local/com/mycila/mycila-log/1.2/mycila-log-1.2.jar:/opt/repo_maven/local/com/mycila/testing/plugins/mycila-testing-spring/2.0-rc1/mycila-testing-spring-2.0-rc1.jar:/opt/repo_maven/local/org/springframework/spring-test/2.5.6/spring-test-2.5.6.jar:/opt/repo_maven/local/com/mycila/testing/plugins/mycila-testing-mockito/2.0-rc1/mycila-testing-mockito-2.0-rc1.jar:/opt/repo_maven/local/org/mockito/mockito-core/1.7/mockito-core-1.7.jar:/opt/repo_maven/local/org/objenesis/objenesis/1.0/objenesis-1.0.jar:/opt/repo_maven/local/org/hamcrest/hamcrest-all/1.1/hamcrest-all-1.1.jar:"/>
    <property name="java.vm.specification.version" value="1.0"/>
    <property name="conf.dir" value="D:/Recia/java/apache-tomcat-5.5.27/wtpwebapps/ESCOGrouper/WEB-INF/classes/properties"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/usr/lib/jvm/java-6-sun-1.6.0.26/jre"/>
    <property name="sun.java.command" value="/tmp/surefirebooter6789415486905512393.jar /tmp/surefire8118576236597836550tmp /tmp/surefire3543385062700101564tmp"/>
    <property name="java.specification.vendor" value="Sun Microsystems Inc."/>
    <property name="user.language" value="fr"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.6.0_26"/>
    <property name="java.ext.dirs" value="/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/ext:/usr/java/packages/lib/ext"/>
    <property name="sun.boot.class.path" value="/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/resources.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/rt.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/jsse.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/jce.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/charsets.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/lib/modules/jdk.boot.jar:/usr/lib/jvm/java-6-sun-1.6.0.26/jre/classes"/>
    <property name="java.vendor" value="Sun Microsystems Inc."/>
    <property name="localRepository" value="/opt/repo_maven/local/"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://java.sun.com/cgi-bin/bugreport.cgi"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.desktop" value="gnome"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase time="0.434" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectById">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
    <system-err>log4j:WARN No appenders could be found for logger (org.springframework.test.context.TestContext).
log4j:WARN Please initialize the log4j system properly.
</system-err>
  </testcase>
  <testcase time="0.041" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectByIdWithSubjectNotFound">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.047" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findMembershipsImmediate">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.046" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findMembershipsEffective">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.041" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findMembershipsAll">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.04" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findMembershipsWithoutMemberships">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.041" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findListTypes">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.042" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findGroupTypes">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.042" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findGroupsMemberOptinOptout">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.034" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findGroupPrivilegesImmediate">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.031" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findGroupPrivilegesEffective">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.031" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findGroupPrivilegesAll">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.032" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectPrivilegesGroupImmediate">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.031" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectPrivilegesGroupEffective">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.034" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectPrivilegesGroupAll">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.038" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectPrivilegesStemImmediate">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.03" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectPrivilegesStemEffective">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
  <testcase time="0.03" classname="org.esco.grouperui.services.grouper.internal.GrouperServiceApiTest" name="findSubjectPrivilegesStemAll">
    <error message="An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext" type="com.mycila.testing.core.api.TestPluginException">com.mycila.testing.core.api.TestPluginException: An error occured while executing &apos;prepareTestInstance&apos; on plugin &apos;spring&apos;: RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:98)
	at com.mycila.testing.junit.MycilaJunitRunner.createTest(MycilaJunitRunner.java:55)
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:213)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:210)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:44)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at com.mycila.testing.junit.MycilaJunitRunner$2.evaluate(MycilaJunitRunner.java:66)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:62)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.executeTestSet(AbstractDirectoryTestSuite.java:140)
	at org.apache.maven.surefire.suite.AbstractDirectoryTestSuite.execute(AbstractDirectoryTestSuite.java:127)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
Caused by: java.lang.RuntimeException: Failed to load ApplicationContext
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:45)
	at com.mycila.testing.core.TestContextImpl.prepare(TestContextImpl.java:96)
	... 23 more
Caused by: java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:201)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:255)
	at com.mycila.testing.plugin.spring.SpringTestPlugin.prepareTestInstance(SpringTestPlugin.java:42)
	... 24 more
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [spring/grouper-api-strategy.xml]; nested exception is java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:349)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:310)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:143)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:178)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:149)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:212)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:81)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:42)
	at org.springframework.test.context.TestContext.loadApplicationContext(TestContext.java:173)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:197)
	... 28 more
Caused by: java.io.FileNotFoundException: class path resource [spring/grouper-api-strategy.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:143)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
	... 37 more
</error>
  </testcase>
</testsuite>